lib/measures/check_model/measure.rb

```diff
@@ -122,7 +143,27 @@ def run(model, runner, user_arguments)
     # install graphviz and make sure dot is in the system path
     tpm.save_graphviz('shell.dot')
     system('dot shell.dot -Tpdf -o shell.pdf')
-
+    
+    # save the full model
+    tpm.save('full_tpm.json')
+    
+    # create a minimal model
```

Comment:
@rd2 here we just transform the Topolys objects to export the schema we want

Reply:
@macumber thanks for the example code.